crane-cloud/frontend
# Single Database Activity

#### What does this PR do?
It creates a page that has a single database view and also incorporates the database reset and database deletion.

#### How to test this PR?
1. Pull this branch
2. Go to your project and on under the sidebar click "databases"
3. You will be able to see a list to all databases.
4. Click on one and you will be re-directed to the single database view.
5. At the bottom of the page lies the delete and reset button click them and test their functionality
6. **Note** use Henry's account to test this out as there lots of databases you can delete plus `DO RUN THE PR LOCALLY!`

#### Ticket 
https://trello.com/c/N1KD8skR

#### Screenshots
![Screenshot from 2021-03-11 14-52-44](https://user-images.githubusercontent.com/32802973/110823948-a08ff080-82a3-11eb-9a08-cbcb38d38846.png)
![Screenshot from 2021-03-11 15-29-24](https://user-images.githubusercontent.com/32802973/110823966-a554a480-82a3-11eb-8737-d230f5b8501c.png)
![Screenshot from 2021-03-11 19-38-10](https://user-images.githubusercontent.com/32802973/110823973-a685d180-82a3-11eb-833d-b9c8454864e8.png)
